app/validators/status_prerequisite_validator.rb

```diff
@@ -1,15 +1,25 @@
 class StatusPrerequisiteValidator < ActiveModel::EachValidator
   def validate_each(record, attribute, value)
     if value == "scheduled"
-      return true if record.has_draft? && record.scheduled_at.present?
+      return true if can_be_scheduled?(record)
 
       record.errors[attribute] << "#{value}, requires a draft and scheduled_at date to be present"
     end
 
     if value == "submitted_for_2i"
-      return true if record.has_draft? && record.review_requester.present?
+      return true if can_be_summited_for_2i?(record)
 
       record.errors[attribute] << "#{value}, requires a draft and review_requester to be present"
     end
   end
+
+private
+
+  def can_be_scheduled?(record)
+    record.has_draft? && record.scheduled_at.present?
+  end
+
+  def can_be_summited_for_2i?(record)
```

Comment:
There's a typo in the name of this method

Reply:
I cannot spell "submitted"!